src/blueDarkTheme.ts

```diff
@@ -6,88 +6,83 @@ All rights reserved.
-        onSurface: black[50],
-        textPalette: {
-            primary: black[50],
-            secondary: black[200],
-            onPrimary: {
-                light: black[500],
-                main: black[500],
-                dark: white[50],
-            },
-            disabled: black[400],
-        },
-        actionPalette: {
-            active: black[500],
-            background: black[800],
-            disabled: Color(black[300]).alpha(themeOpacity.disabled).string(),
-            disabledBackground: Color(black[200]).alpha(themeOpacity.disabledBackground).string(),
-        },
-        disabled: Color(black[300]).alpha(themeOpacity.disabled).string(),
-        notification: lightBlue[200],
-        opacity: themeOpacity,
-        overrides: {
-            avatar: {
-                background: Color(black[50]).alpha(themeOpacity.actionOpacity).string(),
-            },
-            chip: {
-                flat: {
-                    background: black[500],
-                },
-            },
-            toggleButtonGroup: {
-                checked: {
-                    background: Color(blue[500]).alpha(0.36).string(),
-                },
-            },
+        ...MD3DarkTheme.colors,
+        primary: BLUIColors.primary[80],
+        onPrimary: BLUIColors.primary[20],
+        primaryContainer: BLUIColors.primary[30],
+        onPrimaryContainer: BLUIColors.primary[90],
+        primaryNonText: BLUIColors.primary[60],
+        secondary: BLUIColors.primary[80],
+        onSecondary: BLUIColors.primary[20],
+        secondaryContainer: BLUIColors.primary[30],
+        onSecondaryContainer: BLUIColors.primary[90],
+        tertiary: BLUIColors.primary[80],
+        onTertiary: BLUIColors.primary[20],
+        tertiaryContainer: BLUIColors.primary[30],
+        onTertiaryContainer: BLUIColors.primary[90],
+        error: BLUIColors.error[80],
+        onError: BLUIColors.error[20],
+        errorContainer: BLUIColors.error[30],
+        onErrorContainer: BLUIColors.error[90],
+        errorNonText: BLUIColors.error[60],
+        background: BLUIColors.neutral[6],
+        onBackground: BLUIColors.neutral[95],
+        surface: BLUIColors.neutral[6],
+        onSurface: BLUIColors.neutral[95],
+        surfaceVariant: BLUIColors.neutralVariant[30],
+        onSurfaceVariant: BLUIColors.neutralVariant[70],
+        outline: BLUIColors.neutralVariant[60],
+        outlineVariant: Color(BLUIColors.neutralVariant[60]).alpha(0.25),
+        shadow: BLUIColors.primary[0],
+        scrim: BLUIColors.primary[0],
+        inverseSurface: 'rgb(241, 240, 244)',
+        inverseOnSurface: 'rgb(47, 48, 51)',
+        inversePrimary: 'rgb(169, 199, 255)',
+        elevation: {
+            level0: 'transparent',
+            level1: BLUIColors.neutral[10],
+            level2: BLUIColors.neutral[12],
+            level3: BLUIColors.neutral[17],
+            level4: '#29363D',
+            level5: '#2C3941',
```

Comment:
Can you replace elevation level 1 to level 5 with BLUIColors, similar to what you did for the light theme?

<img width="605" alt="image" src="https://github.com/etn-ccis/blui-react-native-themes/assets/8997218/ccb7d2f5-ea66-4301-b435-ba421303881b">